src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -697,11 +697,11 @@ Do you want to continue?</value>
   <data name="UseTheNavigationBarToSwitchContext" xml:space="preserve">
     <value>You can use the navigation bar to switch context.</value>
   </data>
-  <data name="SuppressWithGlobalSuppressMessage" xml:space="preserve">
-    <value>with SuppressMessageAttribute (separate file)</value>
+  <data name="SuppressWithPragma" xml:space="preserve">
+    <value>in Source</value>
   </data>
-  <data name="SuppressWithLocalSuppressMessage" xml:space="preserve">
-    <value>with SuppressMessageAttribute</value>
+  <data name="SuppressWithGlobalSuppressMessage" xml:space="preserve">
+    <value>in Suppression File</value>
```

Comment:
Iirc, the menu text of the suppression fixes were arrived at after some debate :)  There were several reports about users being confused by the names we had picked originally and we changed to the above names per feedback from Lucian, Sri, Mark and others (iirc). It may be worth getting their opinions offline for the new names (i.e. send before and after screenshots).